packages/calling/src/CallingClient/line/index.ts

```diff
@@ -69,6 +69,8 @@ export default class Line extends Eventing<LineEventTypes> implements ILine {
 
   private callManager: ICallManager;
 
+  private serviceData: CallingClientConfig['serviceData'];
```

Comment:
Had to add this private variable over here so that we can access the `GUEST_CALLING` enum value. This is so that we can check for it in the `makeCall()` flow.